code/style.css

```diff
@@ -2,6 +2,491 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+/* Färger
+#62AC7B Main grön
+#437252 Mörk grön
+#34563d Mörkaste grön
+
+#FFCA41 Gul
+#FFD366 Ljjus gul
+#FEF6DF Ljusaste gul
+
+#0F1B12 Mörk grån/svart
+#BABCBB Ljus grå
+*/
+
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: montserrat, sans-serif;
+}
+
+
+/* Header */
+
+header {
+  position: absolute;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  z-index: 1;
+
+}
+
+.logo {
+  width: 220px;
+  margin: 0px 8px;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-self: center;
+  width: 30px;
+  height: 30px;
+  margin: 20px 30px;
+
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #0F1B12;
+  border-radius: 5px;
+}
+
+/* Navbar desktop */
+
+nav {
+  display: flex;
+  justify-content: space-around;
+
+  box-sizing: border-box;
+  padding: 10px;
+  display: none;
+
+}
+
+nav a {
+  box-sizing: border-box;
+  padding: 20px;
+  color: #0F1B12;
+
+  font-size: 20px;
+  font-weight: 600;
+  text-transform:uppercase;
+  letter-spacing: 1px;
+  text-decoration: none;
+}
+
+nav a:hover {
+  background-color: #FFCA41;
+  color: #fff;
+  border-radius: 10px;
+}
+
+
+.mainSection {
+  display: flex;
+  flex-direction: column;
+}
+
+/* Media queries for navbar */
+@media only screen and (min-width: 992px) {
+  nav {
+    display: flex;
+  }
+
+  .hamburger {
+    display: none;
+  }
+}
+
+
+/* hero */
+.heroPic img {
+  width: 100%;
+  background-position: center;
+  background-repeat: none;
+  background-size: cover;
+}
+
+.hero-ctn {
+  text-align: center;
+  width: 100%;
+}
+
+.heroText {
+  position: relative;
+  box-sizing: border-box;
+  font-size: 35px;
+  padding: 30px;
+  margin-top: 20px; 
+}
+
+
+/* Media queries for hero text */
+@media only screen and (min-width: 768px) {
+
+  .heroText {
+    font-size: 50px;
+  }
+} 
+
+/* Media queries for hero text desktop */
+@media only screen and (min-width: 992px) {
+
+  .heroText{
+    position: absolute;
+    z-index: 1;
+    top: 25%;
+    font-size: 50px;
+    text-shadow: 5px 5px  #ffffff;
+    color: #0F1B12;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+}
```

Comment:
Since you've already used this specific media query above then you can add this `.heroText` up there!